modules/get-started/pages/licensing.adoc

```diff
@@ -1,17 +1,13 @@
 = Enterprise Licensing 
 :description: Learn how to purchase and apply an Enterprise Edition license.
 
-Anyone can use the Certified and Community connectors, but you need an Enterprise Edition license to use Enterprise connectors.
+Anyone can use the Certified and Community connectors, but you need a valid Enterprise Edition license to use Enterprise connectors. 
 
-== Purchase an Enterprise Edition license
-
-You can purchase and download a license from the https://www.redpanda.com/upgrade[Redpanda website^].
+You can either https://www.redpanda.com/upgrade[upgrade to an Enterprise Edition license^], or http://redpanda.com/try-enterprise[generate a trial license key^] that's valid for 30 days.
```

Comment:
Maybe put this under a 'Get an enterprise license' heading? It's a user goal that may be searched for.

Reply:
Updated